### DISCUSSION TITLE
Scale image but keep its quality

### DISCUSSION THREAD

**Discussion by mostafasalloumm** [fabricjs/fabric.js#7936] [Q&A] [answered]
Hello, I am trying to scale images to fit the canvas but when I scale it down it loses its quality(resolution) 
is there a way to scale down the image but keep its quality?

**Comment by melchiar** [comment, marked as answer]
Yes! Check out resize filters.

http://fabricjs.com/docs/fabric.Image.filters.Resize.html
http://fabricjs.com/lanczos-webgl
